### ISSUE TITLE
[DQT] Define field tab - cannot select a Visit in Visits filter

### ISSUE THREAD

**Issue by laemtl** [aces/Loris#6583]
In DQT > Define field tab, the checkboxes in the menu dropdown are not selectable.
The checkbox reacts to the click so the click event may be fired twice.  
Observed in Chrome 75.0.3770.80 and Firefox 76.0.1.

**Comment by AlexandraLivadas** [comment]
@laemtl I observed the same issue when testing today. I think that this behavior makes sense and is therefore intended. But I agree with you that there should be a visual indication that the `Visit` checkbox is disabled until you select a field.

**Comment by christinerogers** [comment]
let's file this under Improvements/Cleanup --
cc @maltheism can this nicety be added to the DQT renovation roadmap?

**Comment by ridz1208** [comment]
@maltheism I don't believe this was part of the redesign was it ? if so please close the issue
